app/cafe.py

```
@@ -0,0 +1,19 @@
+from app.errors import (
+    NotWearingMaskError, NotVaccinatedError, OutdatedVaccineError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor don`t have vaccine!")
+        elif datetime.date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("Vaccine expired")
+        elif visitor["wearing_a_mask"] is False:
```

Issue: The current check only raises NotWearingMaskError if 'wearing_a_mask' is explicitly False. According to the requirements, you should also raise this error if the key 'wearing_a_mask' is missing from the visitor dictionary. Please update your condition to handle both cases as required by the checklist.